### PR TITLE
Fix the lack of isblank() with VMS C

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Review by t-j-h — APPROVED** [review]
not the way I would implement the macro but as this is for VMS I don't mind :-)

**Review by paulidale — APPROVED** [review]
Looks good.  The macro is interesting but fine for a test case.

File: test/ctype_internal_test.c [hunk 1]
```diff
@@ -13,6 +13,16 @@
 #include <ctype.h>
 #include <stdio.h>
 
+/*
+ * Even though the VMS C RTL claims to be C99 compatible, it's not entirely
+ * so far (C RTL version 8.4).  For the sake of these tests, we therefore
+ * define our own.
+ */
+#if defined(__VMS) && __CRTL_VER <= 80400000
+# include <string.h>
```
Comment (richsalz) [inline test/ctype_internal_test.c]:
```
static inline int isblank(int n) { return n == ' ' || n == '\t'; }
```

Comment (snhenson) [inline test/ctype_internal_test.c]: Or ossl_inline

**Review by richsalz — APPROVED** [review]

**Comment by levitte** [comment]
The macro turned out to be incorrect, since it returns true for `\0`...

Please reconfirm

File: test/ctype_internal_test.c (hunk 1 above)
Comment (levitte) [inline test/ctype_internal_test.c]: For a local function in a test program, making it inline is a bit over the top, wouldn't you say?  ;-)

**Comment by levitte** [comment]
Ah, I believe @richsalz did reconfirm...

**Comment by levitte** [comment]
Merged. 12bd06cd6b7a67ae6b4f582b085555f62fbea93a

**Pull request closed without merging**